backend/src/main/java/de/lowani/backend/controller/UserController.java

```diff
@@ -0,0 +1,89 @@
+package de.lowani.backend.controller;
+
+
+import de.lowani.backend.api.User;
+import de.lowani.backend.entities.UserEntity;
+import de.lowani.backend.service.UserService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import static de.lowani.backend.controller.UserController.USER_CONTROLLER_TAG;
+import static javax.servlet.http.HttpServletResponse.*;
+import static org.springframework.http.ResponseEntity.ok;
+import static org.springframework.util.MimeTypeUtils.APPLICATION_JSON_VALUE;
+
+@Tag(name = USER_CONTROLLER_TAG, description = "Provides CRUD operations for an User")
+@Api(
+        tags = USER_CONTROLLER_TAG
+)
+@CrossOrigin
+@RestController
+@RequestMapping("/user")
+@Getter
+@Setter
+public class UserController {
+
+    public static final String USER_CONTROLLER_TAG = "User";
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping(produces = APPLICATION_JSON_VALUE)
+    @ApiResponses(value = {
+            @ApiResponse(code = SC_NO_CONTENT, message = "No users found")
+    })
+    public ResponseEntity<List<User>> getAllUsers(){
+        List<UserEntity> allEntities = userService.findAll();
+        if (allEntities.isEmpty()) {
+            return ResponseEntity.noContent().build();
+        }
+        List<User> users = map(allEntities);
+        return ok(users);
+    }
+
```

Comment:
You have a POST with consumes=APPLICATION_JSON_VALUE, but your request body is empty!

Reply:
No, I should have deleted it before pushing, would have been the next step